modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,286 @@
+----
++
+or
++
+[source,terminal]
+----
+$ oc get infrastructure cluster -o json | jq -r '.status.infrastructureName'
+----
+
+.. Identify the
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster OIDC DNS:
++
+--
+[source, terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name> | grep -i 'OIDC'
+----
+
+Save the output from both commands. You will use this information in future steps within this procedure.
+
+--
+. Create and manage AWS Identity and Access Management (IAM) roles and policies:
++
+.. Create the AWS IAM policy required for the ALBO, which can be found link:https://github.com/openshift/aws-load-balancer-operator/blob/61b55feef26e8dfd9d1b2a3300f80ba572f98d58/docs/prerequisites.md#option-2-using-aws-cli[here].
++
+.. Create the AWS IAM policy required for the AWS Load Balancer Controller (ALBC), which can be found link:https://github.com/openshift/aws-load-balancer-operator/blob/61b55feef26e8dfd9d1b2a3300f80ba572f98d58/docs/install.md#option-2-using-the-aws-cli[here].
++
+.. Create an AWS IAM role and assign the ALBO and ALBC AWS IAM policies as definined in the steps above. For more information on creating AWS IAM roles, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html[Creating IAM roles]. For more information on adding AWS IAM policies to AWS IAM roles, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html[Adding and removing IAM identity permissions].
+
+.. Assign the following trust policy to the newly-created AWS IAM role. For more information on assigning trust policies to AWS IAM roles, see link:https://aws.amazon.com/blogs/security/how-to-use-trust-policies-with-iam-roles/[How to use trust policies with IAM roles].
+
++
+--
+[source,json]
+----
+{
+"Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Federated": "arn:aws:iam::{AWS-AccountNo}:{Cluster's OIDC Endpoint}"<1>
+
+            },
+            "Action": "sts:AssumeRoleWithWebIdentity",
+            "Condition": {
+                "StringEquals": {
+                    "{Cluster's OIDC Endpoint}:sub": [
+                        "system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-operator-controller-manager",
+                        "system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-operator-controller-cluster"
+                    ]
+                }
+            }
+        }
+    ]
+}
+----
+<1> Replace '{AWS_AccountNo}' with your AWS account number and '{Cluster_OIDC_Endpoint}' with the OIDC DNS identified in step 1b.
+
+[NOTE]
+====
+Do not include the 'https' portion of the OIDC DNS url when replacing '{Cluster_OIDC_Endpoint}' with the OIDC DNS identified in
+step 1b.  Only the alphanumeric information that follows the '/' within the URL is needed.
```

Comment:
```suggestion
step 1b.  Only the alphanumeric information that follows the `/` within the URL is needed.
```
- Same comment as above re hardcoding step numbers in the procedure. Revise to remove.
- Remove hard line break before line 136
- Put literals in bacticks